README.md

```diff
@@ -1124,13 +1124,25 @@ condition](#safe-assignment-in-condition).
   # good
   names.each { |name| puts name }
 
+  Proc invokation shorthand when that is the only operation of a block
```

Comment:
This is pretty unclear. I'm OK with have a rule suggesting the use of &:something when possible, but it should have crystal clear wording and a separate bullet point.